### DISCUSSION TITLE
Getting page view counts displayed

### DISCUSSION THREAD

**Discussion by pragdave** [vuepress-theme-hope/vuepress-theme-hope#1542] [Q&A] [answered]
I have Waline up and running in Leancloud. Comments work fine, and pageviews work if I add tags manually:

~~~ html
Pageviews: <i id="/posts/active/2022-02-23-the-interview.html" class="waline-visitor-count"></i>
~~~

However, if I don't have the explicit tag, and instead put  `visitor: true` in the frontmatter, it doesn't seem to have any effect.

My goal is to have a page view count on every blog page. Am I missing a configuration step?

**Comment by Mister-Hope** [comment]
It may have some issues here (I haven't tested yet) The article info is provided by @mr-hope/components, so I missed that.

If there are some issues here, I will fix it and release a new version in 48 hours.

**Comment by Mister-Hope** [comment, marked as answer]
v2.0.0-beta.3 should fix this.

**Comment by pragdave** [comment]
And indeed it does. Thank you!
